frontend/www/js/omegaup/ui.ts

```diff
@@ -68,7 +75,11 @@ export function formatString(
   });
 }
 
-export function displayStatus(message: string, type: string): void {
+export function displayStatus(args: {
```

Comment:
último truco:

```suggestion
export function displayStatus({
  message,
  type,
  autoHide,
}: {
```

así no se necesita usar `args`!

Reply:
Nice!